app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -40,13 +43,10 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   // useCallBack here prevents unnecessary rerenders due to changing function identity
   const handleLocationSelect = useCallback(
     async (e: any, id: number) => {
+      flowPath !== 'hybrid' ? e.preventDefault() : null;
```

Comment:
I find these inline ternary expressions to be more difficult to parse vs. the more verbose `if` / `else`, but I've not felt strongly enough to mention its use in past PRs. But I think if we're not even using the `else` portion, I'd feel a bit more compelled to suggest a traditional `if`.

```suggestion
      if (flowPath !== 'hybrid') {
        e.preventDefault();
      }
```

Personally I'd be open to disabling the lint rule to re-allow one-line `if`s as well, if the verbosity is a problem.

```suggestion
      if (flowPath !== 'hybrid') e.preventDefault();
```

Reply:
It's definitely easier to find `if` when scanning a document rather than a ternary. I don't necessarily have an opinion about a single line, so I'm cool with either way. 

Happy to make this change!